app/models/choice.rb

```diff
@@ -2,6 +2,8 @@ class Choice < ActiveRecord::Base
   belongs_to(:answer, :inverse_of => :choices, :touch => true)
   belongs_to(:option, :inverse_of => :choices)
```

Comment:
Any reason to use the old rocket syntax? Should i change the new line i added to use it also?